Content.Client/UserInterface/Controls/RadialMenu.cs

```diff
@@ -229,6 +252,69 @@ public RadialMenuTextureButton()
         OnButtonUp += OnClicked;
     }
 
+    /// <inheritdoc />
+    protected override void Draw(DrawingHandleScreen handle)
+    {
+        var texture = TextureNormal;
+
+        if (texture == null)
+        {
+            TryGetStyleProperty(StylePropertyTexture, out texture);
+            if (texture == null)
+            {
+                return;
+            }
+        }
+        // draw texture
+        handle.DrawTextureRectRegion(texture, PixelSizeBox);
+
+        // draw sector where space that button occupies actually is
+        var pX = -Position.X + Parent!.Width / 2;
+        var pY = -Position.Y + Parent.Width / 2;
+
+        var containerCenter = new Vector2(pX, pY) * UIScale;
+
+        const float singleSegmentSize = MathF.Tau / 32;
+
+        var controlSegmentSize = AngleSectorTo - AngleSectorFrom;
+        var segCount = (int)(controlSegmentSize / singleSegmentSize) + 1;
+
+        // CHANGE TO STACKALLOC AND MOVE THIS STUFF TO TOOLBOX!111
+        var buffer = new Vector2[segCount * 2];
+
+        if (Parent is not RadialContainer container)
+        {
+            return;
```

Comment:
following part is highly dependent on having radius from container so we will have to deal with not working at all if we are in invalid container.